README.md

```diff
@@ -23,20 +25,35 @@ GordianCosigner-Catalyst is currently a late alpha. It should not be used for pr
 
 At current, this Catalyst repo is only used for our iOS release, but we hope to eventually use it for the MacOS release as well. (A security lock on the MacOS camera currently prevents us from doing so.)
 
-## Installation
+## Installation Instructions
 
 **Gordian Cosigner** is available for testing from Testflight [here](https://testflight.apple.com/join/sJTaoUsM).
 
-## Usage
+## Usage Instructions
+
+**Gordian Cosigner** supports the middle step of multisig signing, after a multisig has been Initiated, while it is being Cosigned, and before it is Finalized. Usually, it will be used by a cosigner holding one of the keys for a multisignature before handing it back to another user for finalization.
+
+### Preparing for Usage
+
+To prepare **Gordian Cosigner** for usage, you must first create a copy of the multisig that is being used. This is done by defining an account containing all of the cosigners in the multisig. The user of **Gordian Cosigner** will typically have a private key or seed for his own cosigner element, and then will typically be given xpubs or QR codes for other users' cosigner elements. When he combines these into an account, it should match the multisig created by other people on their own wallets. (See our [Scenarios page](Docs/Scenarios.md) for more discussion of how **Gordian Cosigner** might be used.)
```

Comment:
Typically the GCS user receives an account from the initiator, like I sent you a 2 of 2 today. The ability to create your own accounts in GCS is a convenience, mostly for ease of testing, not a general practice.

Reply:
I've revised the discussion of the "create" testing methods.